kubernetes/samples/scripts/domain-lifecycle/startCluster.sh

```diff
@@ -0,0 +1,120 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a WebLogic cluster in a domain by patching
+  'spec.clusters[<cluster-name>].serverStartPolicy' attribute of the domain
```

Comment:
should there be a period "."?  e.g 'spec.clusters.[<cluster-name>].serverStartPolicy'

Reply:
This was discussed previously in https://github.com/oracle/weblogic-kubernetes-operator/pull/2002#discussion_r511157328. Ryan had suggested to use the exact syntax spec.clusters[?(clusterName="<cluster-name>")] and I had asked if we can use JSON path like syntax for better readability. 

Shall I change it to `spec.clusters[?(clusterName="<cluster-name>")].serverStartPolicy` if you want the exact syntax?

Reply:
I personally don't like the exact syntax since customers rarely use json expressions in every day communication.  Up to you if you want to insert the 'period'.

Reply:
Left it to original value as discussed.